source/usage-examples/find.txt

```diff
@@ -4,13 +4,15 @@ Find Multiple Documents
 
 .. default-domain:: mongodb
 
-You can query for multiple documents in a collection with
-``collection.find()``. The ``find()`` method uses a query document that you
-provide to match the subset of the documents in the collection that match the
-query. If you don't provide a query document (or if you provide an empty
-document), MongoDB returns all documents in the collection. For more
-information on querying MongoDB, see our
-:doc:`documentation on query documents </fundamentals/crud/query-document>`.
+You can query for multiple documents in a collection with the
+``collection.find()`` method. The ``find()`` method uses a query
+document that contains one or more query operators to determine the
+documents to include in the result set. If you don't provide a query
```

Comment:
```suggestion
document  to determine what to include in the result set. Your query document may contain query operators. Query operators are keywords that begin with a ``$`` that let you specify complex query behavior. If you don't provide a query
```

Issue: I don't believe a query document must contain a query operator ({"_id":1} I believe is a valid query document). I think query operators are optional. 

Suggestion: I feel it could be helpful to define the term query operator. 

Sources for suggested definition above:

[Operator Definition
](https://docs.mongodb.com/manual/reference/glossary/#std-term-operator)
[Query Operator Definition](https://docs.mongodb.com/manual/reference/operator/)

This suggestion applies to both pages

Reply:
Well the query document is the BSON we usually call "filter". And to have `{"_id":1}`, you'd need to use `$eq`, which is a query operator.

Reply:
Discussed synchronously. Edit clarified. Agree Query document does not necessarily contain update operator.